app/assets/javascripts/passengers/passengers.js

```diff
@@ -19,4 +11,16 @@ $( document ).on("turbolinks:load", function() {
       }
     });
   });
+
+  $('#passenger_permanent').change(function(){
+    var expirationField = $('.verification-expires');
+    var permanent = $(this).is(':checked')
+    expirationField.prop('disabled', permanent);
+    if(permanent) { expirationField.val('') }
+  });
+
+  $('#passenger_eligibility_verification_attributes_verifying_agency_id').change(function(){
```

Comment:
Whew, what a name.

Reply:
I mean it was either that or *gasp* make my own ID when one already existed.. I don't know.

Reply:
Yeah, I think this is fine; it's just a mouthful. The only other thing that came to mind was something like:

```
$("[id$='verifying_agency_id']")
```

But I don't think exchanging wordiness for more arcane symbols is a move in the positive direction.